tests/test_yaml.py

```diff
@@ -131,3 +107,79 @@ def test_empty_yaml():
 
 
 # can you use mutiple docs? http://pyyaml.org/wiki/PyYAMLDocumentation#LoadingYAML
+
+
+class StringType(ConfigBase):
+    def __init__(self, *args, **kwargs):
+        self._type = str
+        super(StringType, self).__init__(*args, **kwargs)
+
+
+class ProjectDict(ConfigBaseDict):
+    def __init__(self, *args, **kwargs):
+        self._dict_fields = {
+            'github': {
+                'class': StringType,
+                'required': True,
+            },
+        }
+        super(ProjectDict, self).__init__(*args, **kwargs)
+
+
+class ProjectList(ConfigBaseList):
+    def __init__(self, *args, **kwargs):
+        self._list_items_class = ProjectDict
+        self._min_items_required = 1
+        super(ProjectList, self).__init__(*args, **kwargs)
+
+
+class BaseConfig(ConfigBaseDict):
+    def __init__(self, *args, **kwargs):
+        self._dict_fields = {
+            'projects': {
+                'class': ProjectList,
+                'required': True,
+            },
+            'notifications': {
+                'class': ProjectList,
+                'required': True,
+            },
+        }
+        super(BaseConfig, self).__init__(*args, **kwargs)
+
+
+class ProjectLoader(ConfigLoader):
+    config_root_class = BaseConfig
+
+
+def test_multiple_invalid_lines():
+    text = """\
+deadbeef: deadbeef
+alist:
+    - one
+    - two
+    - three"""
+
+    config = ProjectLoader(text)
+    assert not config.is_valid()
+
+    expected_error = """\
+deadbeef: deadbeef
+# ^
```

Comment:
We could certainly do that, or else indent every line by 2, except our comments.  Whichever you want.